src/app/pages/guide-list/guide-list.html

```diff
@@ -1,8 +1,8 @@
 <header class="docs-primary-header">
-  <h1>Guides</h1>
+  <h1 focusOnNavigation>Guides</h1>
```

Comment:
It seems weird to me that the auto-focused element and the element the skip link takes you to are not the same